### DISCUSSION TITLE
Execute code before API call [Aug 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6162] [Discord Threads]
> 🙋‍♂️ **girlazo**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported.  
#  
> 🙋‍♂️ **girlazo**  
#### Ok! Thank you very much for the early response <:girlazLove:747926413755285625>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1126081646114906182)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
